### PR TITLE
[codelab] Update TLSR9518 OT Guide Codelab

### PR DESCRIPTION
**Pull request by pulin1103**
Here are  some updates:   
    1.Replace TLSR9518 with B91;
    2.Take new photos of boards and adjust the text size in the images;
    3.Removed the content related to serial port conversion, and the device now supports USB-CDC-ACM.
Please let me know if there are any issues with the content.

### CODE REVIEW

**Comment by Vyrastas** [comment]
@jrhodie please also take a look at this and review.

**Review by jrhodie — APPROVED** [review]
LGTM

File: site/en/codelabs/telink-openthread-hardware/index.lab.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ layout: scrolling
 
 ---
 
-# Build a Thread Network with the TLSR9518 Development Kit and OpenThread
+# Build a Thread Network with the B91 Development Kit and OpenThread
```
Comment (Vyrastas) [inline site/en/codelabs/telink-openthread-hardware/index.lab.md:15]: Sometimes you use "Development Kit" and other times "Development Board".  Suggest it be "Board" throughout the codelab, unless there's a specific reason to use "kit".

File: site/en/guides/index.md [hunk 2]
```diff
@@ -51,6 +51,9 @@ Dive right into hardware, where you will learn how to:
    href="https://openthread.io/codelabs/silabs-openthread-hardware/">Try the Silicon Labs Hardware
     Codelab</a>
 
+<a class="button button-primary"
```
Comment (Vyrastas) [inline site/en/guides/index.md:54]: Thanks for adding this, I forgot about it!

File: site/en/codelabs/telink-openthread-hardware/index.lab.md [hunk 3]
```diff
@@ -87,31 +85,29 @@ To accomplish these tasks, the Linux machine requires two available USB ports an
 
 ### Serial Port Connection and Terminals
 
-It is recommended to be familiar with establishing connections between devices using a serial port converter cable.
+You can directly plug the device into the USB port of Linux machine.
```
Comment (Vyrastas) [inline site/en/codelabs/telink-openthread-hardware/index.lab.md:88]: of the Linux machine.

File: site/en/codelabs/telink-openthread-hardware/index.lab.md [hunk 4]
```diff
@@ -329,26 +325,18 @@ $ sudo ./bdt 9518 wf 0 -i bin/ot-cli-ftd.bin
...
 Duration: 3:00
 
-To control the FTD Joiner device through the CLI, connect a USB to TTL Serial Convertor Cable to the following pins on the TLSR9518 development board：
-
-| Name  | Pin                 |
-| :---- | ------------------- |
-| RX    | PB3 (pin 15 of J34) |
-| TX    | PB2 (pin 18 of J34) |
-| GND   | GND (pin 23 of J50) |
+As shown in the picture, directly plug the FTD Joiner into the USB port of the computer.
```
Comment (Vyrastas) [inline site/en/codelabs/telink-openthread-hardware/index.lab.md:335]: of the Linux machine.

**Review by Vyrastas — CHANGES_REQUESTED** [review]
Thanks for doing the update, just a few nits

File: site/en/codelabs/telink-openthread-hardware/index.lab.md (hunk 3 above)
Comment (pulin1103) [reply site/en/codelabs/telink-openthread-hardware/index.lab.md:88]: Added in the updated commit.

File: site/en/codelabs/telink-openthread-hardware/index.lab.md (hunk 4 above)
Comment (pulin1103) [reply site/en/codelabs/telink-openthread-hardware/index.lab.md:335]: Added in the updated commit.

File: site/en/codelabs/telink-openthread-hardware/index.lab.md (hunk 1 above)
Comment (pulin1103) [reply site/en/codelabs/telink-openthread-hardware/index.lab.md:15]: Your advice is very accurate, so I replaced most of "Development Kit" with "Development Board". However, I retained "kit" in the Telink B91 Development Kit section, as this subsection provides an introduction to the development kit."

**Review by jwhui — APPROVED** [review]

**Review by jrhodie — COMMENTED** [review]
LGTM.

**Pull request merged**